tools/frr-reload.py

```diff
@@ -421,30 +421,6 @@ def save_contexts(self, key, lines):
                     newlines.append(line)
             lines = newlines
 
-        # More fixups in user specification and what running config shows.
-        # "null0" in routes must be replaced by Null0.
-        if (
-            key[0].startswith("ip route")
-            or key[0].startswith("ipv6 route")
-            and "null0" in key[0]
-        ):
-            key[0] = re.sub(r"\s+null0(\s*$)", " Null0", key[0])
```

Comment:
This is still needed I suppose.

Reply:
Hi, thanks for review.
It will make more complex. In frr.conf with things of 'null0', but running config with 'Null0'. 
In reload.py's comparing between running and file configuration, it will create more things of "no ....".